Python/Product/PythonTools/PythonTools/Extensions.cs

```diff
@@ -375,22 +375,27 @@ public static class Extensions {
 
         internal static PythonProjectNode GetProjectFromFile(this IServiceProvider serviceProvider, string filename) {
```

Comment:
I'd like to make it clear that this is not going to return the project if the file isn't opened in an editor.  Maybe add a summary comment or rename the method to make it more obvious.